python/BUILD

```diff
@@ -14,3 +14,16 @@
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
+
+# Python2 is a dependency of rules_docker due to its dependency on
+# containerregistry. To run builds on environments in which 'which python'
+# points to python3, you can use the py_runtime target below.
```

Comment:
We should also verify that this works with the `build_python_zip` bazel flag.

Reply:
I'm not too familiar with the build_python_zip flag. I ran the tests in the repo using these flags:
--python_top=//python:py_runtime_2 --build_python_zip
and the tests worked as expected (some of our tests seem to not be compatible with build_python_zip flag, but for unrelated reasons). Please let me know if you think we need to perform other checks to verify this works as you expect.

Reply:
Thanks for testing with the flag. I think this is sufficient.